clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -544,8 +564,8 @@ private void resetSequenceNumbers() {
         this.partitionsWithUnresolvedSequences.clear();
     }
 
-    synchronized void requestEpochBumpForPartition(TopicPartition tp) {
-        epochBumpRequired = true;
+    synchronized void requestIdempotentEpochBumpForPartition(TopicPartition tp) {
```

Comment:
Should we leave a comment here for what method to use for transactional producers?

Reply:
There's nothing that does the exact same thing so idk if mentioning the needEpochBumpTrigger etc would make sense

Reply:
I guess I just meant to specify not to use for transactional producers as well?